packages/flutter_tools/lib/src/asset.dart

```diff
@@ -60,6 +60,8 @@ class AssetBundle {
      int result = await _validateFlutterYamlManifest(manifest);
      if (result != 0)
        return result;
+    } else {
+      return 0;
```

Comment:
This is the else branch of if (manifest != null). Basically if a project doesn't have a manifest we move on.